src/Tasks/Microsoft.NET.Build.Tasks.UnitTests/GivenAResolvePackageDependenciesTask.cs

```diff
@@ -40,36 +40,16 @@ public static IEnumerable<object[]> ItemCounts
                 {
                     new object[] {
                         "dotnet.new",
-                        new int[] { 110, 2536, 1, 846, 73 },
-                        true
-                    },
-                    new object[] {
-                        "dotnet.new",
-                        new int[] { 110, 0, 0, 846, 0 },
-                        false
-                    },
-                    new object[] {
-                        "simple.dependencies",
-                        new int[] { 113, 2613, 1, 878, 94 },
-                        true
+                        new int[] { 110, 2536, 1, 845, 75 },
```

Comment:
Did you figure out why some of these numbers are changing?

Reply:
This unit-test was already broken in release/7.0.2xx. This PR is not affecting the results of this tests.